posthog/models/event/sql.py

```diff
@@ -199,15 +206,22 @@
 )
 
 # This table is responsible for reading from events on a cluster setting
-DISTRIBUTED_EVENTS_TABLE_SQL = lambda: EVENTS_TABLE_BASE_SQL.format(
-    table_name="events",
+DISTRIBUTED_EVENTS_TABLE_SQL = lambda name="events": EVENTS_TABLE_BASE_SQL.format(
```

Comment:
I would just rename the table, but that breaks previous migrations.

This way we can keep previous migrations and call it from the tests with the new events table name.

It's dirty, suggestions are very welcomed.

Reply:
I had to change the above because many tests depend on the events table.

What I have done right now (which I'm still not happy about) is that the view is only created when running the migrations, meanwhile in the test model we don't create the view and we leave the events table as it was.

I don't like it either because we are not having the same data model in the tests and the actual ClickHouse. I'll probably revert that. 

Maybe we just need to just fix all the tests so that when they perform writes, they are made against the original table and reads are made from the view. Not sure how to approach it.